pkg/controller/common/helpers.go

```diff
@@ -22,9 +24,12 @@ import (
 	ign3_1 "github.com/coreos/ignition/v2/config/v3_1"
 	translate3_1 "github.com/coreos/ignition/v2/config/v3_1/translate"
 	ign3_1types "github.com/coreos/ignition/v2/config/v3_1/types"
-	ign3 "github.com/coreos/ignition/v2/config/v3_2"
-	translate3 "github.com/coreos/ignition/v2/config/v3_2/translate"
-	ign3types "github.com/coreos/ignition/v2/config/v3_2/types"
+	ign3_2 "github.com/coreos/ignition/v2/config/v3_2"
+	translate3_2 "github.com/coreos/ignition/v2/config/v3_2/translate"
+	ign3_2types "github.com/coreos/ignition/v2/config/v3_2/types"
+	ign3 "github.com/coreos/ignition/v2/config/v3_3_experimental"
+	translate3 "github.com/coreos/ignition/v2/config/v3_3_experimental/translate"
+	ign3types "github.com/coreos/ignition/v2/config/v3_3_experimental/types"
```

Comment:
I think instead of moving this to spec 3.3, we should add 3.3 as a separate case for now so the default is still 3.2.

Alternatively, we can move individual items (e.g. ParseAndConvertConfig) to spew out 3.2, such that rendered configs are 3.2